bin/alitv.pl

```diff
@@ -247,6 +247,151 @@ =head1 PARAMETERS
 YML file will be created based on the default template. The name of
 the file will be given for further runs.
 
+=head1 Description of the YML file format
+
+The YML file contains the whole project description. Currently, it
+supports three different sections:
```

Comment:
Generally most of the comments from above apply to this file as well as it contains identical text in some places